src/scss/base/_typography.scss

```diff
@@ -233,6 +233,18 @@ samp { font-family: $font-family-mono; }
     text-align: right;
 }
 
+.text-highlight {
+    color: $red;
+
+    @at-root em#{&} {
```

Comment:
Don't think so.

The above results in `em.text-highlight {` as a selector.  I think yours would be `.text-highlight.em` (Haven't tested, but looks like that would be what it would output❓)

Reply:
You're right. I thought that `em&` would do the same but that errors.

Reply:
Yeh – I think it's because it can't parse it, so you effectively have to escape it using a combination of `@root` and `#{}`